src/charmed_kubeflow_chisme/testing/_unit_tests.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""
+# Testing
+
+Tools for unit or integration testing, such as importable and reusable tests.
+
+## Usage example
+
+```python
+import pytest
+from charm import Operator
+from charmed_kubeflow_chisme.testing import test_leadership_events as leadership_events
+from charmed_kubeflow_chisme.testing import test_missing_image as missing_image
+from ops.model import WaitingStatus
+from ops.testing import Harness
+
+
+@pytest.fixture
+def harness():
+    return Harness(Operator)
+
+
+def test_leadership_events(harness):
+    leadership_events(harness)
```

Comment:
Is there a way to avoid passing `harness` every time?

Reply:
A way to avoid passing it every time would be to import the tests without aliases:
```python
from chisme import test_leadership_events
```
That way when pytest runs the test suite, it will look for functions called test_*, including anything imported. When we have a harness fixture in the charm's unit tests, it will be used by the imported tests.

The downside is, that wouldn't be intuitive as the only 'trace' of that test presence would be the import statement.

Reply:
Understood, better to be more explicit.